metrics/api/src/main/java/io/helidon/metrics/api/KeyPerformanceIndicatorMetricsConfigBlueprint.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+import io.helidon.builder.api.Prototype;
+import io.helidon.config.metadata.Configured;
+import io.helidon.config.metadata.ConfiguredOption;
+import io.helidon.inject.configdriven.api.ConfigBean;
+
+/**
+ * Config bean for KPI metrics configuration.
+ */
+@ConfigBean()
+@Configured(prefix = MetricsConfigBlueprint.METRICS_CONFIG_KEY
```

Comment:
If this is not a root configured component, then the prefix is useless. Prefix does not switch to root...

Reply:
In general there are just too many constants used here. If a string is only used once, I would prefer not to use a constant.

Reply:
I'll trim them back. Allowing config keys to be derived from the method names will help remove the need for some of the constants.